test/unit/vl-grid.test.html

```diff
@@ -116,6 +122,11 @@
   });
 
   suite('vl-column', () => {
+    test('standaard size en max-size wordt gezet indien er attributen geconfigureerd zijn', () => {
```

Comment:
Moet dit niet zijn, als er geen attributen geconfigureerd zijn?

Reply:
Inderdaad, werd reeds aangepast in #80.